ha/core/controllers/pcs/node_controller.py

```diff
@@ -114,7 +119,27 @@ def start(self, nodeid: str) -> dict:
             ([dict]): Return dictionary. {"status": "", "msg":""}
                 status: Succeeded, Failed, InProgress
         """
-        raise HAUnimplemented("This operation is not implemented.")
+        _res = self.nodes_status([nodeid])
+        if isinstance(_res, str):
+            _res = json.loads(_res)
+        _all_node_status = _res.get("msg")
```

Comment:
`_res = json.loads(_res)`
_res is dict do we need to convert it to json ?

Reply:
I think in @controller_error_handler decorator we do return return json.dumps(result) value. as its used for each function result returns as string

Reply:
yes but _res = self.nodes_status([nodeid])

nodes_status not using `@controller_error_handler` it only for plugin exposed outside
nodes_status is not exposed and for only internal use like utility

Reply:
agree, did changes